### DISCUSSION TITLE
今天发现绳网的Web页面被标记成“存在诈骗风险了”

### DISCUSSION THREAD

**Discussion by swt1me** [share121/inter-knot#5279] [其他]
![image](https://github.com/user-attachments/assets/4e8de7ab-4dcf-4de4-81f3-c55443089cf2)
本来向上官网看看结果就发现被标“存在诈骗风险了”

**Comment by WuxuRiver** [comment]
给新艾利都治安局盯上了
![IMG_20240913_192814](https://github.com/user-attachments/assets/31322a19-bd52-4295-8841-12018f7bc485)

**Comment by manyshoes** [comment]
这种网站要备案吗

**Comment by dahetaoa** [comment]
没猜错的话你应该是福建地区的，福建地区的网络是白名单机制，也就是说不在白名单内的网址一律ban掉，要么直接拦截要么重定向到反诈网站

**Reply by WuxuRiver** [reply]
> 没猜错的话你应该是福建地区的，福建地区的网络是白名单机制，也就是说不在白名单内的网址一律ban掉，要么直接拦截要么重定向到反诈网站

不一定，我就是福建的，我这也没问题

**Reply by swt1me** [reply]
不是哦

**Comment by DogOfFurina114514** [comment]
我**没显示**有风险，但**连不上**
![image](https://github.com/user-attachments/assets/7478b883-f2f1-42d2-9dc3-751148883b73)

**Comment by keyuanya** [comment]
我广东，家里移动宽带和数据网络直连都没问题
![1725170579643de9f30837f7d0e58d91350b98fd04a72767852faae9c4d8aeb01166fcb7f60b5 0](https://github.com/user-attachments/assets/be066ce5-6bed-4998-9a56-d2cf81d85d3f)

**Comment by Shadow00624** [comment]
确实，我也遇到了

**Comment by TNCNET** [comment]
这边青海的，一般来说都没问题，能直连。

**Comment by HoshinoHanashi** [comment]
惯性穿越，暂无问题<img src="https://upload-bbs.miyoushe.com/upload/2023/01/18/8481df2cd99d2b00ecb94a92a14c9fe4_7915718047845434249.png" alt="米游兔 OK" width="75" >
